lib/rolodex/processors/swagger.ex

```diff
@@ -183,7 +194,27 @@ defmodule Rolodex.Processors.Swagger do
     |> process_schema_field()
   end
 
-  defp process_schema_field(field), do: Map.take(field, @schema_metadata_keys)
+  defp process_schema_field(field) do
+    field
+    |> Map.take(@schema_metadata_keys)
+    |> put_description(field)
+  end
+
+  ## Helpers ##
+
+  defp collect_required_object_props(props), do: Enum.reduce(props, [], &do_props_collect/2)
+
+  defp do_props_collect({k, %{required: true}}, acc), do: [k | acc]
+  defp do_props_collect(_, acc), do: acc
+
+  defp set_required_object_props([], object), do: object
```

Comment:
Why is this case needed?

Reply:
The idea is that we only want to add the `:required` attribute to the map when there's one or more require properties. An empty list (`required: []`) is not valid according to Swagger's docs.